theseus/geometry/point_types.py

```diff
@@ -91,7 +91,7 @@ def exp_map(
         if tangent_vector.ndim != 2 or tangent_vector.shape[1] != 2:
             raise ValueError("Tangent vectors of Point2 should be 2-D vectors.")
 
-        Vector._exp_map_jacobian(tangent_vector, jacobians)
+        Vector._exp_map_jacobian_impl(tangent_vector, jacobians)
```

Comment:
Should there be functions in Point2/3 for logmap calling `Vector._log_map_jacobian_impl`?

Reply:
Not necessary since `Point2.log_map()` and `Point3.log_map()` calls `Vector.log_map()`

Reply:
Ah right. For exp map you needed it explicitly here for the tangent vector size check.

Reply:
Yes, we checked the sizes [here](https://github.com/facebookresearch/theseus/blob/e487c03917ad19747c2336bdd39f8dbe1d8b70d5/theseus/geometry/point_types.py#L91).